samples/msteams-chat-with-your-data/README.md

```diff
@@ -0,0 +1,61 @@
+# Chat With Your Data
```

Comment:
Please refer to the PR below for your reference, including images and sample setup details with format.
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/1322